### ISSUE TITLE
Readme for repo template

### ISSUE THREAD

**Issue by willscott** [fidlabs/allocator-tooling#3]
there should be some text in a readme that helps allocators undertand how to use the allocator tooling when it's set up

**Comment by willscott** [comment]
this should be done as an update to the readme in https://github.com/fidlabs/allocator-template

**Comment by clriesco** [comment]
Should this readme file be overwritten in every allocator repo?

**Comment by willscott** [comment]
nope. allocators may have started customizing their local instances of readmes. this is most relevant for the template and for new allocators when contents are first pushed
